cookbook-react/src/recipes/lists/valparaiso/List/styles.module.scss

```
@@ -0,0 +1,29 @@
+$alabaster-tertiary: #DFDFDF;
+$alabaster: #F7F7F7;
+$black: #000;
+
+.list-item {
+  border-radius: 6px;
+  height: 50px;
+  padding: 20px 20px 20px 25px;
```

_suggestion: I think it's clearer._
```suggestion
  padding: 20px;
  padding-left: 25px;
```